psydac/feec/multipatch/examples/h1_source_pbms_conga_2d.py

```diff
@@ -11,23 +11,24 @@
 
 from sympde.expr.expr import LinearForm
 from sympde.expr.expr import integral, Norm
-from sympde.topology  import Derham
+from sympde.topology import Derham
 from sympde.topology import element_of
 
-
-from psydac.api.settings        import PSYDAC_BACKENDS
+from psydac.api.settings import PSYDAC_BACKENDS
 from psydac.feec.multipatch.api import discretize
-from psydac.feec.pull_push      import pull_2d_h1
+from psydac.feec.pull_push import pull_2d_h1
 
-from psydac.feec.multipatch.fem_linear_operators        import IdLinearOperator
-from psydac.feec.multipatch.operators                   import HodgeOperator
-from psydac.feec.multipatch.plotting_utilities          import plot_field
+from psydac.feec.multipatch.fem_linear_operators import IdLinearOperator
+from psydac.feec.multipatch.operators import HodgeOperator
+from psydac.feec.multipatch.plotting_utilities import plot_field
 from psydac.feec.multipatch.multipatch_domain_utilities import build_multipatch_domain
-from psydac.feec.multipatch.examples.ppc_test_cases     import get_source_and_solution
-from psydac.feec.multipatch.utilities                   import time_count
+from psydac.feec.multipatch.examples.ppc_test_cases import get_source_and_solution_OBSOLETE
```

Comment:
why call this function OBSOLETE if it is still used ?

Reply:
I looked into it again, and apparently I was starting to separate the examples in  the 'ppc_test_cases' function into their own functions depending of the test equation. (see e.g. 'get_source_and_solution_h1') I had the feeling that 'ppc_test_cases' is not very readable, but apparently I didn't go fully through as I wasn't familiar with most examples in there.